storage/object/api-cli/using-api-call-list.mdx

```diff
@@ -21,75 +21,83 @@ In this page we index all available Scaleway Object Storage API requests.
 </Message>
 
 Status:
-  - Feature available: <Icon name="validate" /> 
-  - Feature coming in 2022: ⌛
-
+  - Feature available: ✅
+  - Feature coming in 2023: ⌛
+  - Deprecated feature: ❗
 ## Common Operations
 
 | Feature | Description | Status |
 | :------------ | :---- | :- |
-|  [GetService](/storage/object/api-cli/common-operations/#getservice) | Returns a list of all buckets | <Icon name="validate" /> |
+|  [GetService](/storage/object/api-cli/common-operations/#getservice) | Returns a list of all buckets | ✅ |
 
 
 ## Bucket Operations
 
 | Feature | Description | Status |
-| :------------ | :---- | :- |
-|  [DeleteBucket](/storage/object/api-cli/bucket-operations/#deletebucket) |  Deletes bucket | <Icon name="validate" /> |
-|  [DeleteBucketCors](/storage/object/api-cli/bucket-operations/#deletebucketcors) |  Deletes the [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) configuration of a bucket | <Icon name="validate" /> |
-|  [DeleteBucketLifecycle](/storage/object/api-cli/bucket-operations/#deletebucketlifecycle)|  Deletes the [lifecycle configuration](/storage/object/api-cli/lifecycle-rules-api/) of a bucket | <Icon name="validate" /> |
-|  [DeleteBucketPolicy](/storage/object/api-cli/bucket-operations/#deletebucketpolicy) |  Deletes the policy of a specified bucket | <Icon name="validate" /> |
-|  [DeleteBucketTagging](/storage/object/api-cli/bucket-operations/#deletebuckettagging) |  Deletes the tag(s) of a bucket | <Icon name="validate" /> |
-|  [DeleteBucketWebsite](/storage/object/api-cli/bucket-operations/#deletebucketwebsite) |  Deletes the bucket website configuration of a specified bucket | <Icon name="validate" /> |
-|  [GetBucket](/storage/object/api-cli/bucket-operations/#getbucket-(list-objects)) |  Lists objects | <Icon name="validate" /> |
-|  [GetBucketAcl](/storage/object/api-cli/bucket-operations/#getbucketacl) |  Returns the bucket ACL | <Icon name="validate" /> |
-|  [GetBucketCors](/storage/object/api-cli/bucket-operations/#getbucketcors) |  Returns the [CORS](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing) configuration of a bucket | <Icon name="validate" /> |
-|  [GetBucketLifecycle](/storage/object/api-cli/bucket-operations/#getbucketlifecycle)|  Returns the [lifecycle configuration](/storage/object/api-cli/lifecycle-rules-api/) of a bucket | <Icon name="validate" /> |
-|  [GetBucketLocation](/storage/object/api-cli/bucket-operations/#getbucketlocation) |  Returns the region where the bucket is | <Icon name="validate" /> |
+| :------------ | :---- | :---- |
+|  [CreateBucket](/storage/object/api-cli/bucket-operations/#putbucket) |  Creates bucket | ✅ |
```

Comment:
Maybe, we also can change naming `PutBucket` -> `CreateBucket` in this page : /storage/object/api-cli/bucket-operations/#putbucket